walkthroughs/week-4-tdd/project/src/test/java/com/google/sps/FindMeetingQueryTest.java

```diff
@@ -64,6 +66,12 @@ public void optionsForNoAttendees() {
     Collection<TimeRange> expected = Arrays.asList(TimeRange.WHOLE_DAY);
 
     Assert.assertEquals(expected, actual);
+
+    // If there are other events scheduled in the day
+    actual = query.query(Arrays.asList(Events.events), request);
```

Comment:
Additional test should be in separate test case.
This ensures that the test is testing one thing at a time.